React/Views/RCTWebView.m

```diff
@@ -73,6 +73,11 @@ - (void)setURL:(NSURL *)URL
   [_webView loadRequest:[NSURLRequest requestWithURL:URL]];
 }
 
+- (void)setHTML:(NSString *)HTML
+{
+  [_webView loadHTMLString:HTML baseURL:nil];
```

Comment:
Could just add it as another prop I guess. 




My only issue is that I don't think baseURL makes sense without the HTML prop, just in the same way I think it would be good to require the HTML prop OR the URL prop. I'm not sure React supports this kind of "dependent" prop validation?

On Tue, Mar 31, 2015 at 6:02 PM, Spencer Ahrens <notifications@github.com>
wrote:

>> @@ -73,6 +73,11 @@ - (void)setURL:(NSURL *)URL
>>    [_webView loadRequest:[NSURLRequest requestWithURL:URL]];
>>  }
>>  
>> +- (void)setHTML:(NSString *)HTML
>> +{
>> +  [_webView loadHTMLString:HTML baseURL:nil];
> Thoughts on baseURL support in the future?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/facebook/react-native/pull/542/files#r27499311